sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientTests.cs

```diff
@@ -112,11 +113,12 @@ public void StartRecognizeContentFromUriRespectsTheCancellationToken()
         {
             var client = CreateInstrumentedClient();
             var fakeUri = new Uri("http://localhost");
+            var options = new RecognizeOptions { ContentType = ContentType.Pdf };
 
             using var cancellationSource = new CancellationTokenSource();
             cancellationSource.Cancel();
 
-            Assert.ThrowsAsync<TaskCanceledException>(async () => await client.StartRecognizeContentFromUriAsync(fakeUri, default, cancellationSource.Token));
+            Assert.ThrowsAsync<TaskCanceledException>(async () => await client.StartRecognizeContentFromUriAsync(fakeUri, options, cancellationSource.Token));
```

Comment:
Do we need to specify content type for URI methods?

Reply:
No, we don't. There was no need to modify this test. I'll revert it back.